README.md

```diff
@@ -17,6 +17,7 @@ Pulls multiple podcast feeds (RSS) and republishes as a common feed, properly so
 | feed             | FM_FEED           |                 | single feed, overrides config             |
 | update-interval  | UPDATE_INTERVAL   | `1m`            | update interval, overrides config         |
 | telegram_chan    | TELEGRAM_CHAN     |                 | single telegram channel, overrides config |
+| telegram_server  | TELEGRAM_SERVER   | `https://api.telegram.org` | telegram bot api server, can be replaced with [the local one](https://core.telegram.org/bots/api#using-a-local-bot-api-server) to raise audio file upload limit from 50Mb to 2000Mb |
```

Comment:
if we add the section about tg local tingy as suggested above the desctiption can be short with a reference to tho section